test/node/conversation-labels.js

```diff
@@ -0,0 +1,111 @@
+'use strict';
+
+const assert = require('assert');
+const Circuit = require('../../circuit-node');
+const config = require('./config.json');
+const helper = require('./helper');
+Circuit.logger.setLevel(Circuit.Enums.LogLevel.Error);
+
+let client;
+let addedLabelsHT = {};
+describe('Authentication', () => {
+
+    it('should login using Client Credentials', async () => {
```

Comment:
The login test is part of authentication, so that should not be tested here. 
You should create a "before" function that logs in at the beginning on this labels test suite.
See https://github.com/circuit/circuit-sdk/blob/master/test/browser/directcall.js#L16 which has such a before function, but in your case you just need to logon.